src/CurrencyCard/CurrencyCard.js

```diff
@@ -1,23 +1,17 @@
 import React, { useState, useEffect } from 'react'
-import { Route } from 'react-router-dom'
 import './CurrencyCard.scss';
 
-const CurrencyCard = () => {
-  const [currencyCard, setCurrencyCard] = useState()
-
+const CurrencyCard = ({ newAmount, userCurrency, userAmount, newCurrency, exchangeRate }) => {
   return (
       <section className='currency-card'>
-        <h1>This would be an individual Currency Card</h1>
-        
+        <h1>{newAmount}</h1>        
```

Comment:
I still need to organize this data in a way that is useful to a user. But at least all the data is finally there!